src/index.js

```diff
@@ -0,0 +1,50 @@
+import './style.css';
+
+const todos = [
+  {
+    index: 1,
+    description: 'Finish awesome books mobile version',
+    completed: false,
+  },
+
+  {
+    index: 2,
+    description: 'Grocery Shopping',
+    completed: true,
+  },
+
+  {
+    index: 3,
+    description: 'Go for a 30-minute run',
+    completed: false,
+  },
+
+  {
+    index: 4,
+    description: 'Review a pull request submitted by a colleague',
+    completed: true,
+  },
+
+  {
+    index: 5,
+    description: 'Do laundry',
+    completed: false,
+  },
+];
+
+const toDosSection = document.getElementById('todos');
+const displayToDos = todos.forEach((todo) => {
+  const div = document.createElement('div');
+  div.className = 'todo';
+  const pTag = document.createElement('p');
+  pTag.className = 'todo-description';
+  pTag.innerHTML = todo.description;
+  const completedBtn = document.createElement('input');
+  completedBtn.type = 'checkbox';
+  completedBtn.value = todo.completed;
+  div.appendChild(completedBtn);
+  div.appendChild(pTag);
+  toDosSection.append(div);
+});
+
+displayToDos();
```

Comment:
Error in the console (`displayToDos` is not a function)

![image](https://user-images.githubusercontent.com/78251927/228379883-81a3b6c3-6f30-4c0a-b95b-d199991e549e.png)

Your code 
```JavaScript
const displayToDos = todos.forEach((todo) => {
  . . .
});

displayToDos();
```

I could see that you were trying to use an arrow function here, but I think you are missing the `()` and `=>` after the function name `displayToDos`. 🤗 please let's add them 👍

e.g 👍 
```JavaScript
const displayToDos = () => todos.forEach((todo) => {
  . . .
});

displayToDos();
```